packages/mui-codemod/README.md

```diff
@@ -62,6 +62,14 @@ npx @mui/codemod <transform> <path> --jscodeshift="--printOptions='{\"quote\":\"
 
 ### v5.0.0
 
+#### `date-pickers-moved-to-x`
+
+Rename the imports of date and time pickers from `@mui/lab` to `@mui/x-date-pickers` and `@mui/x-date-pickers-pro`.
+
+```sh
+npx @mui/codemod v5.0.0/date-pickers-moved-to-x <path>
```

Comment:
This codemod impacts v5 but not v5.0.0. Maybe it would have made more sense with

```suggestions
npx @mui/codemod v5.6.0/date-pickers-moved-to-x <path>
```


or

```suggestions
npx @mui/codemod v5/date-pickers-moved-to-x <path>
```